### ISSUE TITLE
How to clear closed files in "Problems"?

### ISSUE THREAD

**Issue by Redfish52** [castwide/vscode-solargraph#87]
When I close my files, the problems is still showing:
![sreenshot](https://user-images.githubusercontent.com/41098503/49490629-9e43be00-f872-11e8-86a1-86ff1cba20fc.png)

**Comment by castwide** [comment]
I'm afraid I haven't been able to reproduce this error. Closing the file automatically clears its problems in all my tests.

Do you receive any error messages, either in the output view or the developer console?

**Comment by Redfish52** [comment]
> Do you receive any error messages, either in the output view or the developer console?
No, I don't have other errors. I will try reinstall the ruby and gems.
